docs/dm/assets.md

```diff
@@ -0,0 +1,170 @@
+# Assets
+
+The asset menu is used for inserting assets into your game. 
+These can be maps, tokens, objects, or anything else you wish to insert. 
+To access this menu, click <font-awesome :icon="['far', 'compass']"/> and click “Assets” from the dropdown menu.
+
+## Asset Manager
+
+![](./asset-manager.png)
+
+If you’re just starting out, you will have no assets. 
+To add them, you will need to enter the Asset Manager by clicking ![](./assets-link.png).
+
+### Creating Folders
+
+![](./asset-manager-create-folder.png)
+
+To create a folder, click the create folder icon.
+A dialogue box should appear where you can write the new folder name.
+With the folder created, you can place your assets into the folder for easier navigation.
+
+### Uploading Files
+
+![](./asset-manager-upload-files.png)
+
+To upload files, click the upload files icon.
+This will open your file system.
+Choose the asset you wish to upload to your campaign.
+Once uploaded, your files will be accessible in-game. 
+
+## Placing Assets
+
+![](./asset-example.gif)
+
+To use your assets, first open the asset tab. 
+Then, open the folder where you have stored your files.
+Last, drag the asset onto the game board. 
+Your asset will now be able to be manipulated.
+
+## Manipulating Assets
+
+You can move your assets by selecting the “select” tool.
+Then you can click and drag the asset.
+
+![](asset-list.png)
```

Comment:
just took a quick look why the build is failing and this image should have a ./ ;)

Reply:
Oh dang, my bad! I'll go ahead and fix the image. To fix the build failing, would I need to add links to these two documents to the doc-links.yaml file? Apologies if this is a pretty basic question, I'm still a little new to git and coding and general